doc/source/conf.py

```diff
@@ -120,7 +122,10 @@
 templates_path = ['../_templates']
 
 # The suffix of source filenames.
-source_suffix = '.rst'
+source_suffix = [
+    '.rst',
+    '.txt',
```

Comment:
required so that the whatsnew files can be included in the toctree.

Reply:
Is there actually a reason they are not *.rst ?